tools/pubsys/src/repo.rs

```diff
@@ -352,10 +352,11 @@ where
     match repo_load_result {
         // If we load it successfully, build an editor and manifest from it.
         Ok(repo) => {
+            let target = "manifest.json".try_into().unwrap();
```

Comment:
Same as other locations.

BTW, if we get clippy going in Bottlerocket CI https://github.com/bottlerocket-os/bottlerocket/issues/1776 then we can add this to make sure we don't panic anywhere:

```rust
#![deny(
    clippy::expect_used,
    clippy::get_unwrap,
    clippy::panic,
    clippy::panic_in_result_fn,
    clippy::panicking_unwrap,
    clippy::unwrap_in_result,
    clippy::unwrap_used
)]
```

Reply:
I wonder tough could have const checks for static strings, so we could construct things like this without runtime error checking...